context/build.gradle

```diff
@@ -1,31 +1,16 @@
 plugins {
-    id "java"
-    id "java-test-fixtures"
+    id "java-library"
     id "maven-publish"
-
-    id "me.champeau.gradle.japicmp"
-    id "me.champeau.jmh"
-    id "ru.vyarus.animalsniffer"
 }
 
 description = 'gRPC: Context'
 
 sourceCompatibility = 1.7
-targetCompatibility = 1.7
```

Comment:
We want to keep this as well.

Reply:
As discussed removed both because that works better

Reply:
These aren't copied to the pom, so removing them works without issue.